### PR TITLE
Jb add relation style parameter

### PR DESCRIPTION
**Pull request by jbutcher21**
update icij to use the new relationship anchor and pointers.   also corrected an error with addresses.

### CODE REVIEW

File: icij_mapper.py [hunk 1]
```diff
@@ -383,13 +395,15 @@ def node2Json(tableName, nodeRecord, nodeDatabase, nodeType):
     argparser.add_argument('-l', '--log_file', default=os.getenv('log_file'.upper(), None), type=str, help='optional statistics filename (json format).')
     argparser.add_argument('-d', '--database', default=os.getenv('database'.upper(), 'ALL'), type=str, help='choose: panama, bahamas, paradise, offshore or all (default=all)')
     argparser.add_argument('-t', '--node_type', default=os.getenv('node_type'.upper(), 'ALL'), type=str, help='choose: entity, intermediary, officer or all (default=all)')
+    argparser.add_argument('-r', '--relationship_style', dest='relationship_style', type=int, default=2, help='styles: 0=None, 1=Legacy linking, 2=Pointers (new for Senzing v1.15)')
```
Comment (antaenc) [inline icij_mapper.py:398]: You don't need to use dest= when the long format option is the same.

File: icij_mapper.py [hunk 2]
```diff
@@ -383,13 +395,15 @@ def node2Json(tableName, nodeRecord, nodeDatabase, nodeType):
...
     argparser.add_argument('-R', '--reload_csvs', default=False, action='store_true', help='reload from csvs, don\'t use cached data')
     args = argparser.parse_args()
     inputPath = args.input_path
     outputFileName = args.output_file
     logFile = args.log_file
     nodeDatabase = args.database.lower() if args.database else None
     nodeType = args.node_type.lower() if args.node_type else None
+    relationshipStyle = args.relationship_style
```
Comment (antaenc) [inline icij_mapper.py:406]: Why create new var? args are in main and globally scoped you can use args.* elsewhere.

File: icij_mapper.py [hunk 3]
```diff
@@ -152,7 +152,7 @@ def processTable(fileDict):
     tableName = fileDict['tableName']
     
     #--these aren't entities
-    if nodeType in ['edges', 'address', 'other']:
+    if nodeType in ['edges', 'other']:  #--'address', go ahead and load addresses as entities
```
Comment (antaenc) [inline icij_mapper.py:155]: Use a tuple instead of a list.

**Review by antaenc — APPROVED** [review]
Couple of small things for consideration.

File: icij_mapper.py (hunk 1 above)
Comment (jbutcher21) [reply icij_mapper.py:398]: Yes, I am slowly getting rid of those.

File: icij_mapper.py (hunk 3 above)
Comment (jbutcher21) [reply icij_mapper.py:155]: Gotcha, I will start using tuples.

**Pull request merged**